src/Codebase/Infrastructure/SourceParser/Parser/PhpParser.php

```diff
@@ -27,7 +27,7 @@ class PhpParser implements ParserInterface
     /**
      * @var string
      */
-    protected const PARSER_EXTENSION = 'php';
+    public const PARSER_EXTENSION = 'php';
```

Comment:
If $type is not necessary it should be reverted.

Reply:
We need it, the same reason that in first comment